adding final group separator option

funnell opened the pull request:
Here is my initial attempt at adding a final group separator option that addresses #209. I took a best guess at where to insert it, let me know what you think. Thanks for your help and guidance!

alphapapa commented:
@funnell Thanks for requesting the review to remind me about this.  I have several Emacs projects and repos, so it's easy for me to overlook updated issues and PRs that need my attention again.  Also, I don't have as much free time to work on these projects as I used to, so please feel free to remind me now and then if necessary.

I've just spent a few hours trying to fix "decay" in the test suite resulting from changes to Org and Emacs in recent versions.  I need the test suite to be working properly again before I can merge any changes like this.  That work is almost done, but not quite complete, so it will be a bit longer before I can merge this.  But looking at the code, I see no reason it can't be merged.  

Please go ahead and squash the commits.  Also, feel free to add a changelog entry in the readme according to the format used, but if you don't, I'll take care of it when I merge.  Thanks.

funnell commented:
@alphapapa I understand completely, thanks for taking the time to maintain this wonderful package and considering my pull request! I've squashed the commits and added a line to the readme.

alphapapa commented:
@funnell The tests shouldn't be affected by this, so I added a missing period and line break and merged to master.  Thanks for your work on this.
